From cd315ee8bb453320fa1ad616cb3ba30364590bff Mon Sep 17 00:00:00 2001
From: Bart Ribbers <bribbers@disroot.org>
Date: Tue, 25 Nov 2025 12:05:27 +0100
Subject: [PATCH] systemd: install service file to the system units directory

Just /usr/lib/systemd isn't valid, it needs to be a system or user unit.
In our case a system unit is the right choice.
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 723c7bf937..9b55cdbc9c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -169,6 +169,6 @@ ki18n_install(po)
 if (INSTALL_SYSTEMD_SERVICE)
     install(FILES
         ${CMAKE_CURRENT_SOURCE_DIR}/plasma-mobile.service
-        DESTINATION ${KDE_INSTALL_SYSTEMDUNITDIR}
+        DESTINATION ${KDE_INSTALL_SYSTEMDUNITDIR}/system
     )
 endif()
-- 
GitLab

